server/emails/verification/email.js

```diff
@@ -0,0 +1,17 @@
+import sgMail from "@sendgrid/mail";
+import sendVerificationTemplate from "./config";
+
+sgMail.setApiKey(process.env.SENDGRID_API_KEY);
+export const sendVerificationEmail = async options => {
+  const { firstName, lastName, userEmail, verificationUrl } = options;
+  const msg = {
+    to: userEmail,
+    from: process.env.EMAIL,
+    subject: "Verify Email",
+    html: sendVerificationTemplate(firstName, lastName, verificationUrl)
+  };
+  const mail = await sgMail.send(msg);
+  return mail;
+};
+
+// export default sendVerificationEmail;
```

Comment:
there is something wrong here, this file has changed. This is not the new email.js. remember both of us worked together to refractor the email folder.